themes/openy_themes/openy_rose/templates/block/block--openy-rose-branding.html.twig

```diff
@@ -18,7 +18,8 @@
 {% block content %}
   {% if site_logo %}
     <a href="{{ path('<front>') }}" title="{{ 'Go to the homepage'|t }}" rel="home">
-      <img src="{{ site_logo }}" alt="{{ 'Site-logo'|t }}" />
+      <img class="logo" src="{{ site_logo }}" alt="{{ 'Site-logo'|t }}" />
+      <img class="aof" src="{{ base_path }}/{{ directory }}/img/aof.svg" alt={{ 'Areas of focus'|t }}"/>
```

Comment:
It doesn't work on the build. We should have absolute path with base URL that will include `/buildID`.